install_heroku_script.sh

```diff
@@ -0,0 +1,9 @@
+cd iframely/
+npm install
+echo "---------------1-------------------"
+cd ../
+cd oauth-proxy/
+npm install
+echo "------------------------------2--------------------------"
+cd ../
+gulp dev
```

Comment:
heroku deployments, `gulp prod` and then the generated build needs to be served.